backend/migrations/2023-03-03-140707_convert_variety_to_text/up.sql

```diff
@@ -0,0 +1,10 @@
+-- Your SQL goes here
+ALTER TABLE seeds DROP COLUMN variety_id;
+DROP TABLE varieties;
+
+-- Add column without not null constraint.
+ALTER TABLE seeds ADD COLUMN variety TEXT;
+-- Then set all varieties to some default value .... 
+UPDATE seeds SET variety = 'Default';
+-- ... before setting not null constraint.
+ALTER TABLE seeds ALTER COLUMN variety SET NOT NULL;
```

Comment:
Variety of seeds can be NULL. This represents that the the variety of the plant_id is used.